util/src/keys/geth_import.rs

```diff
@@ -93,6 +94,46 @@ pub fn import_geth_keys(secret_store: &mut SecretStore, geth_keyfiles_directory:
 	Ok(())
 }
 
+
+/// Gets the default geth keystore directory.
+///
+/// Based on https://github.com/ethereum/go-ethereum/blob/e553215/common/path.go#L75
+pub fn keystore_dir() -> PathBuf {
+	
+	#[cfg(unix)]
+	fn data_dir(mut home: PathBuf) -> PathBuf {
+		home.push(".ethereum");
+		home  
+	}
+	
+	#[cfg(target_os = "macos")]
```

Comment:
As per the linked https://github.com/ethereum/go-ethereum/blob/e553215/common/path.go#L75 instead of `macos` it should be `darwin` for Mac, i.e

`#[cfg(target_os = "darwin")]`